docs/source/description.md

```diff
@@ -0,0 +1,166 @@
+etc.) are not yet implemented! :::
+
+The hpge copper support structure consists of three components:
+
+- a copper rod supporting each string (shown at the top of the rendering), these
+  have names:
+
+```
+hpge_support_copper_string_support_structure_string_{STRING}
+```
+
+where `{STRING}` is the string number,
+
+- a triangular copper support for each string (or "tristar") named
+
+```
+hpge_support_copper_tristar_{SIZE}_string_{STRING}
+```
+
+where `{SIZE}` is the size of the string (`small`,`medium`, `large`, or
+`xlarge`).
+
+- copper rods for each HPGe detector which have names:
+
+```
+hpge_support_copper_string_{STRING}_cu_rod_{IDX}
+```
+
+where `{IDX}` is an index of the rod.
+
+:::{tip}
+
+To select all copper parts in _remage_ you can use a wildcard
+`hpge_support_copper.*` and similarly to select all copper rods, tristar or
+string support structures. :::
+
+## Top plate
+
+The `top` assembly consists of the top plate holding the CC4 electronics,
+currently this is a single physical volume called `top_plate`.
+
+This is shown on the left figure of the rendering below.
+
+```{subfigure} AB
+:subcaptions: above
+
+:::{image} ./images/top_plate.png
+:height: 400px
+:alt: Copper top plate
+:::
+
+:::{image} ./images/fibers.png
+:height: 400px
+:alt: Fiber shrouds.
+:::
+
+```
+
+&nbsp;
+
+## Fiber's and SiPMs
+
+The fiber shroud's for the LAr readout are shown in the right figure above.
+
+As mentioned in {doc}`runtime-cfg` there are two modes for the optical fibers,
+either individual fibers or a segmented option.
+
+In both cases the fiber volumes are divided into 4 parts:
+
+- an outer TPB coating,
+- two layers of cladding,
+- the fiber core.
+
+The optical fiber system consists of a large number of physical volumes, to
+enable concise _remage_ macros the names are first prefixed with:
+
+- `fiber_inner`: for the inner barrel fibers (inner cylinder),
+- `fiber_outer`: for the outer barrel fibers.
+
+Next the name contains an identifier of the part of the fiber either, `tpb`,
+`cladding1`, `cladding2` or `core`.
+
+The rest of the names give further information on the fiber, and uses the length
+of the fiber, whether it is part of the lower bend for the outer barrel and the
+fiber index to obtain a unique physical volume name.
+
+:::{tip} For more users it is expected to use wildcards to select groups of
+optical fibers. :::
+
+The SiPMs are named after the **detector name** (ie `S001`), and there are also
+physical volumes of a wrapping around the SiPMs with names `{NAME}_wrap` (where
+`{NAME}` is the SiPM name).
+
+## Wavelength shifting reflector
+
+The array is emersed in liquid argon (physical volume named `lar`) and
+surrounded by a tetratex and TPB lined copper foil for reflecting scintillation
+light (WLSR).
+
+This is shown as the outer cylinder below, which shows all the main components
+of the experiment. The WLSR is the shown as the outer grey component. :::{image}
```

Comment:
this does not work with a formatting like this

there are also more problems like this in this file that need to be fixed